dask_cuda/cuda_worker.py

```diff
@@ -9,6 +9,7 @@
 from toolz import valmap
 from tornado.ioloop import IOLoop
 
+import dask
 from distributed import Nanny
 from distributed.config import config
```

Comment:
```suggestion
```

Failing due to a linting issue:
> dask_cuda/cuda_worker.py:14:1: F401 'distributed.config.config' imported but unused

Reply:
The canonical way to do this is to `import dask` and then use `dask.config.get`

Reply:
Ah, I see that this is already done, and that you're just suggesting removing this.  Woot.